### PR TITLE
NEW: dataloaf tabulate viz

### PR DESCRIPTION
**Pull request by lizgehret**
This PR adds a visualizer for the DataLoafPackageDirFmt that is an output of the ANCOM-BC wrapper method within q2-composition. This visualizer provides tab separated views for each dataframe containing the log-fold change (lfc), standard error (se), P and Q values, and W scores.

### CODE REVIEW

**Comment by lizgehret** [comment]
Here's the latest [output viz](https://view.qiime2.org/visualization/?type=html&src=13679256-82d8-43b7-9978-7338ac603ee9) using some EMP data, as a reference for what the current output looks like.

**Comment by lizgehret** [comment]
To test out this viz, you can download [this QZV file from Google Drive](https://drive.google.com/file/d/1h9N-u4EdO3BXfTMpj0lucb6XW2FQihim/view?usp=share_link), and run `qiime tools view` locally.

File: q2_composition/plugin_setup.py [hunk 1]
```diff
@@ -104,3 +103,11 @@
             ex.ancombc_multi_formula_with_reference_levels)
     }
 )
+
+plugin.visualizers.register_function(
+    function=q2_composition.tabulate,
+    inputs={'dataloaf': FeatureData[DifferentialAbundance]},
```
Comment (ebolyen) [inline q2_composition/plugin_setup.py:109]: Maybe call this parameter `data` instead? Dataloaf won't mean much to end-users.

File: q2_composition/plugin_setup.py [hunk 2]
```diff
@@ -104,3 +103,11 @@
...
+    parameters={},
+    name='Tabulate',
```
Comment (ebolyen) [inline q2_composition/plugin_setup.py:111]: A few more words would be good here.

File: q2_composition/plugin_setup.py [hunk 3]
```diff
@@ -104,3 +103,11 @@
...
+    description='',
```
Comment (ebolyen) [inline q2_composition/plugin_setup.py:112]: We need some kind of description implying what you would do with this and why

**Review by ebolyen — CHANGES_REQUESTED** [review]
Code looks good to me, we just need to flesh out the `visualizers` plugin registration a bit!

**Review by ebolyen — APPROVED** [review]

**Pull request merged**